style.css

```diff
@@ -0,0 +1,345 @@
+  }
+
+  body {
+    margin: 0;
+    font-family: Inter, sans-serif;
+  }
+
+  header {
+    height: 50px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  nav {
+    display: flex;
+    width: 95%;
+    height: 50px;
+    margin-left: auto;
+    margin-right: auto;
+    align-items: center;
+    justify-content: space-between;
+    background-color: white;
+    position: fixed;
+  }
+
+  .nav-h3 {
+    margin-left: 15px;
+    text-decoration: none;
+    color: #172b4d;
+  }
+
+  .intro-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 40px;
+    background-image: url('./Assets/Images/header-illsutration-mobile@2x.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .intro-wrappers {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    color: #172b4d;
+    font-size: 40px;
+    line-height: 40px;
+    font-weight: 700px;
+    margin: 0;
+  }
+
+  .intro-wrappers1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    color: #36b37f;
+    font-size: 40px;
+    margin: 0;
+  }
+}
+
+.intro-wrapper-p {
+  width: 327px;
+  height: 164px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+}
+
+.intro-wrapper-ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  background-color: transparent;
+  width: 100%;
+  padding: 0;
+}
+
+.intro-wrapper-ul-li {
+  margin: 5px 10px;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 0;
+}
+
+.w-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+}
+
+.works-ul {
+  list-style: none;
+  width: 90%;
+  margin: 0;
+}
+
+.list-item {
+  list-style: none;
+  width: 90%;
+  margin: 15px 0;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 5px;
+}
+
+.top-notch {
+  width: 100%;
+  border-radius: 8px 8px 0 0;
+  margin-top: 200px;
+  background-color: white;
+}
+
+.top-notch-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.top-tag {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.top-notch-content-ul {
+  display: flex;
+  align-items: center;
+  width: 90%;
+  justify-content: space-between;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: white;
+  padding: 0;
+  list-style: none;
+}
+
+.top-notch-content-ul-li {
+  font-size: 12px;
+  border-radius: 5px;
+  border: 0 solid #ebf0ee;
+  background-color: #ebf0ee;
+  padding: 5px 10px;
+}
+
+.top-notch-content-a {
```

Comment:
- [ ] Please make sure that all your `buttons` follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468))

Reply:
Hi @ridwanediallo, I tried to look for the effect of the button on hovers and pressed but could not see it and I also realised a mobile phone has no hover effect, please enlighten me on this. Nothing on figma